web/wp-content/themes/dctx-v2/template-parts/content-password-protected.php

```diff
@@ -0,0 +1,21 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-pxozot.php, but found phpcs-scan-pXOzot.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).